django/applications/catmaid/models.py

```diff
@@ -876,21 +881,22 @@ def get_subclass_links_qs():
             return num_linked_ci < self.value
         elif self.cardinality_type == 3:
             # Exactly n for each sub type
-            subclass_links_q = get_subclass_links()
+            subclass_links_q = get_subclass_links() # type: ignore
+                                                    # FIXME: Probably a serious bug - I can't find where this function is supposed to be defined
```

Comment:
Thanks, good catch! It should be `get_subclass_links_qs()` defined a few lines above. Will fix it in `dev`.